WEB-MAC-CHAR/src/main/resources/static/js/voteManager.js

```diff
@@ -0,0 +1,24 @@
+export default class voteManager {
+    constructor(dayTime) {
+      this.slotBox = document.querySelector('.slot_box');
+    }
+    
+    init() {
+      this.slotBox.addEventListener('click', this.dayTimeVote);
+      Array.from(document.getElementsByClassName("player_status")).forEach(entry => {
+        entry.innerHTML = "";
+      });
+    }
+    
+    dayTimeVote(e) {
```

Comment:
If you need `e.target`, how about this?
```js
dayTimeVote({target}) {
    ...
}
```
so, we just use target in this function.